src/client/pages/Checkout/CheckoutPayment/index.tsx

```diff
@@ -19,7 +19,11 @@ export const Checkout = () => {
     return (
       <>
         <LoadingPage loading />
-        <CheckoutErrorModal isVisible={error !== undefined} onRetry={onRetry} />
+        <CheckoutErrorModal
```

Comment:
Nit: `isVisible` prop can be replaced with conditional rendering
```
!!error && <CheckoutErrorModal ... />
```

Reply:
I thought conditional rendering for modals was somehow gonna be less performant than letting the modal itself handle that logic.

However, now I can't find any such references.

Still, we should either do conditional rendering OR pass `isVisible` prop, right?

Reply:
Forgot it's modal, let's keep original version